Sources/PaymentsCore/api/EligibilityAPI.swift

```diff
@@ -1,11 +1,13 @@
 import Foundation
 
+/// API that return merchant's elgibility for diferent payment methods: Venmo, PayPal, PayPalCredit, PayLater & credit card
 public class EligibilityAPI {
 
     private let graphQLClient: GraphQLClient
     private let apiClient: APIClient
     private let coreConfig: CoreConfig
 
+    /// Initialize the elgibility api to check for payment methods eligiblity
```

Comment:
```suggestion
    /// Initialize the eligibility API to check for payment methods eligibility
```

Need to also add a parameters object of coreConfig passed into the init. If you cmd + click you'll see an option in Xcode that says "Add Documentation" that will auto fill out the format including params for documentation